dev-workflow-ui-web-test/src_test/ch/ivyteam/ivy/project/workflow/webtest/WebTestStartsIT.java

```diff
@@ -200,4 +231,125 @@ void startableIcons() {
     starts.searchGlobal("HomePageTestData.ivp");
     $(By.id("startsForm:projectStarts:0:processStartIcon")).shouldBe(visible).shouldHave(cssClass("si-house-1"));
   }
+
+  @Test
+  void filterFromUrl() {
+    var testValue = "thisistest";
+    openView("starts.xhtml");
+    $(By.id("startsForm:projectStarts:globalFilter")).shouldBe(empty);
+    openView("starts.xhtml", Map.of("q", testValue));
+    $(By.id("startsForm:projectStarts:globalFilter")).shouldHave(value(testValue));
+  }
+
+  @Test
+  void globalFilterDoesNotPersistAcrossPageReloads() {
+    openView("starts.xhtml");
+
+    typeFilterAndWait("makeAdmin");
+
+    openView("home.xhtml");
+    openView("starts.xhtml");
+
+    $(By.id("startsForm:projectStarts:globalFilter")).shouldBe(empty);
+  }
+
+  @Test
+  void urlParametersWorkForGlobalFilter() {
+    openView("starts.xhtml");
+
+    openView("starts.xhtml", Map.of("q", "urlFilter"));
+    $(By.id("startsForm:projectStarts:globalFilter")).shouldHave(value("urlFilter"));
+
+    openView("home.xhtml");
+    openView("starts.xhtml");
+    $(By.id("startsForm:projectStarts:globalFilter")).shouldBe(empty);
```

Comment:
this is already tested with the `globalFilterDoesNotPersistAcrossPageReloads` test